doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -0,0 +1,322 @@
+<br>
+Click on **Next** to proceed.
+
+<br>
+
+#### Cellular Setup:
+
+- Set the Cellular Configuration to **Enabled** and add the corresponding APN if you are using Cellular as the backhaul. 
+
+> APN stands for Access Point Name. The APN is assigned by your wireless service provider.
+
+{{< figure src="012_Cellular_Configuration.png" alt="Cellular Configuration " >}}
+
+
+- Click on **Next**, and the Cellular Authentication window appears. You can leave the default setting as-is. 
+
+
+{{< figure src="013_Cellular Authentication.png" alt="Cellular Authentication" >}}
+
+Click on **Next** again to proceed. 
+
+<br>
+
+#### Remote Management
+
+If you prefer Remote Management, you can **Enable** the feature and provide the necessary Account Key.  To know more about how to configure Remote Management, you can refer to this **[software guide](https://www.multitech.com/documents/publications/software-guides/s000727%20-%20mPower%20Edge%20Intelligence%20Conduit_AEP_software_guide.pdf).**
+
+
+{{< figure src="014_Remote_Management.png" alt="Remote Management" >}}
+
+Click on **Next** to proceed.
+
+
+#### 
+
+<br>
+#### HTTP/HTTPS Access
+
+The device allows only secure access to its Web UI. This panel defines how the device handles HTTP traffic like, automatically redirecting HTTP requests to the device's secure HTTPS port, etc. You can choose **LAN** or **WAN** as desired.
+<br>
+
+{{< figure src="015_Access_Configuration.png" alt="Access Configuration" >}}
+
+Click on **Next** to proceed.
+
+#### 
+
+<br>
+#### Bootloader Protection
+
+This feature enables password authentication to access the device bootloader. It is disabled by default.
+
+
+{{< figure src="016_Bootloader_Protection.png" alt="Bootloader Protection" >}}
+
+
+> The Bootloader Protection settings depend on the previous u-boot configuration and are preserved when the device is reset to factory defaults.
+
+
+Click on **Finish**.
+#### 
+
+<br>
+#### Adding {{%tts%}} Server Address in the Gateway
+<br>
+
+- Click on "**LoRaWAN**" in the menu on the left. It opens the Gateway's configuration page.
+
+<br>
+
+{{< figure src="017_001_Gateway_Menu_LoRaWAN.png" alt="Gateway Menu" >}}
+<br>
+
+- Select the mode as "**Packet Forwarder**" 
+{{< figure src="018_Packet_Forwarder_Gateway.png" alt="Packet Forwarder Gateway" >}}
+
+<br>
+
+- Select your **frequency plan** (EU868 in this case.)
+
+{{< figure src="019_Frequency_Plan.png" alt="Frequency Plan" >}}
+
+- Scroll down and add:
+
+|---------------------------------------------------------------------|-----------------------------------------|
+| {{%tts%}} URL provided to you as the **Server Address**. | `thethings.example.com` |
+| Upstream Port                                                       | 1700                                    |
+| Downstream Port                                                     | 1700                                    |
```

Comment:
![image](https://user-images.githubusercontent.com/507540/71548116-b03d5500-29cf-11ea-9eff-eab4f17c9125.png)

Formatting is lost. Check.